website/content/docs/connect/cluster-peering/usage/create-cluster-peering.mdx

```diff
@@ -0,0 +1,834 @@
+---
+layout: docs
+page_title: Establish Cluster Peering Connections
+description: >-
+  Generate a peering token to establish communication, export services, and authorize requests for cluster peering connections. Learn how to create, list, read, check, and delete peering connections.
+---
+
+# Establish cluster peering connections
+
+This page details the process for establishing a cluster peering connection between services deployed to different datacenters. You can interact with Consul's cluster peering features using the HTTP API, the CLI, the UI, or the Kubernetes controller. While the specific steps vary slightly for each, the overall process for establishing a cluster peering connection consists of the following steps:
+
+1. Create a peering token in one cluster.
+1. Use the peering token to establish peering with a second cluster.
+1. Export services between clusters.
+1. Create intentions to authorize services for peers.
+
+Cluster peering between services cannot be established until all four steps are complete.
+
+For more information about using Kubernetes with cluster peering, including required Helm values, required CRDs, and other configuration options, refer to [cluster peering on Kubernetes](/consul/docs/connect/cluster-peering/usage/k8s)
+
+## Requirements
+
+To establish a cluster peering connection, your deployments must meet several requirements. For additional guidance on required configuration, refer to the [cluster peering technical specifications](/consul/docs/connect/cluster-peering/configuration).
+
+<Tabs>
```

Comment:
@david-yu 

curious if you think there's a better order for this (currently, API, CLI, UI, Kubernetes). we should look at heap to see which one is more used... but from a gut feeling i think (Kubernetes, CLI, API, UI). what do you think?

Reply:
Since we have a page dedicated to Kubernetes, should we just link to it? it has a more comprehensive set of instructions imo

https://consul-cyxww49ps-hashicorp.vercel.app/consul/docs/connect/cluster-peering/usage/k8s#guidance-for-new-users

Reply:
Yes I would say Kubernetes, CLI, API and UI is the correct order. 

@im2nguyen as far as why Jeff added these changes, he added them because it sounded like it was not obvious to a user what certain steps are needed in K8s versus VMs. I would say this page with the tabs serves more as a reference guide in a sense, where as the K8s one is an end to end example.

However, there is merit to also say you can do an all in one for each platform on a single page with a bunch of tabs. 
As a nit, do we know if the examples across all tabs relates to the same workload across all tabs? As in do the intentions, etc look mostly the same?